Ports/cavestory/package.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port='cavestory'
+version='2.6.5-1'
+files=(
+    "https://github.com/nxengine/nxengine-evo/archive/refs/tags/v${version}.zip v${version}.zip a3a1a802eda46f9922065a35624ba94e3823649a7e71012e479ed91fcdb50255"
+    'https://www.cavestory.org/downloads/cavestoryen.zip cavestoryen.zip aa87fa30bee9b4980640c7e104791354e0f1f6411ee0d45a70af70046aa0685f'
+)
+auth_type='sha256'
+depends=(
+    'libjpeg'
+    'libpng'
+    'SDL2'
+    'SDL2_image'
+    'SDL2_mixer'
+    'SDL2_ttf'
+)
+workdir="nxengine-evo-${version}"
+launcher_name='Cave Story'
+launcher_category='Games'
+launcher_command='/usr/local/bin/nxengine-evo'
+icon_file='platform/switch/icon.jpg'
+
+post_fetch () {
+  mkdir -p ${workdir}/build
+}
+
+build () {
+  cd ${workdir}/build
+  cmake -DCMAKE_BUILD_TYPE=Release .. 
+  make
+}
+
+pre_install () {
+  mv CaveStory/Doukutsu.exe ${workdir}/
+  mv CaveStory/data/* ${workdir}/data/
+  rm -rf CaveStory
+  export PATH=$PORT_BUILD_DIR/${workdir}:$PATH
+  cd ${workdir}
+  nxextract  
+}
+
+install () {
+  cd ${workdir}/build 
+  make prefix=/usr/local DESTDIR=$SERENITY_INSTALL_ROOT install
+}
+
+uninstall () {
+  cd ${workdir}/build
```

Comment:
This uninstall doesn't do anything.

Reply:
This is not resolved - `uninstall` should be removed.